modules/hostap/src/supp_main.c

```diff
@@ -98,7 +98,6 @@ static const struct wifi_mgmt_ops mgmt_ap_ops = {
 	.ap_enable = supplicant_ap_enable,
 	.ap_disable = supplicant_ap_disable,
 	.ap_sta_disconnect = supplicant_ap_sta_disconnect,
-	.ap_bandwidth = supplicant_ap_bandwidth,
```

Comment:
please make a separate commit for hostap, and I guess the missing definition `supplicant_ap_bandwidth` is what is incomplete? and this is an accidental change?